### ISSUE TITLE
Cryptic car parameter names make car support painfully annoying

### ISSUE THREAD

**Issue by vntarasov** [commaai/openpilot#124]
I'm looking at changes in 0.3.6, and my eyes hurt:
```
-  a_y = v_ego**2 * angle_steers * deg_to_rad / (CP.steerRatio * CP.wheelBase)
+  a_y = v_ego**2 * angle_steers * deg_to_rad / (CP.sR * CP.l)
```

I need CarParams interface to have legible and descriptive names.

**Comment by espes** [comment]
Will merge a PR that renames them 👍

**Comment by Stormwind99** [comment]
Is this the merge you've been waiting for?

https://github.com/commaai/openpilot/commit/1ad9cc8c67f2c784907acdd6316a16061987b71d

**Comment by vntarasov** [comment]
Yes, fixed in 1ad9cc8c67f2c784907acdd6316a16061987b71d 👍
